vsintegration/src/FSharp.Editor/CodeLens/LineLens.fs

```diff
@@ -350,10 +350,29 @@ type internal FSharpCodeLensService
             let! taggedText, navigation = lens.TaggedText
             let textBox = new TextBlock(Width = 500., Background = Brushes.Transparent, Opacity = 0.0, TextTrimming = TextTrimming.None)
             DependencyObjectExtensions.SetDefaultTextProperties(textBox, formatMap.Value)
-            textBox.Inlines.Add (Documents.Run Settings.CodeLens.Prefix)
+
+            let prefix = Documents.Run Settings.CodeLens.Prefix
+            prefix.Foreground <- SolidColorBrush(Color.FromRgb(153uy, 153uy, 153uy))
+            textBox.Inlines.Add (prefix)
+
             for text in taggedText do
+
+                let properties = layoutTagToFormatting text.Tag
+                let properties =
+                    if Settings.CodeLens.UseColors
+                    then
+                        match properties.ForegroundBrush with
+                        | :? SolidColorBrush as b ->
+                            let c = b.Color
+                            // Change to correct gray color
+                            if c.R = c.G && c.R = c.B then properties.SetForeground(Color.FromRgb(153uy, 153uy, 153uy)) else properties
```

Comment:
We should make this customizable.

Reply:
Sure, but how? Is there a color picker control we can use? (I've no experience with WPF.)